src/actions/hitherJobs.ts

```diff
@@ -7,8 +7,8 @@ export const addHitherJob = (job: HitherJob): RootAction => ({
     job: job
 })
 
-export const updateHitherJob = (args: { jobId: string, update: HitherJobUpdate}): RootAction => ({
+export const updateHitherJob = (args: { clientJobId: string, update: HitherJobUpdate}): RootAction => ({
```

Comment:
Use clientJobId rather than the (python) jobId to monitor jobs on the client side. This is because if the job fails to create on the python side, it will not have a job ID, only a client job ID.